extensions.tex

```diff
@@ -79,12 +79,44 @@ \subsection{Beta Distribution Bag-of-words}
   \subsection{Grid Searching}
   \label{sec:grid_search}
 
-    \subsubsection{RNN}
-    \label{sec:rnn_grid_search}
+  \subsubsection{RNN}
+  \label{sec:rnn_grid_search}
 
-\subsection{Ensemble}
-\label{sec:ensemble}
+  \subsection{Ensemble}
+  \label{sec:ensemble}
+  
+  \subsection{Translation}
+  \label{sec:translation}
+ To improve the results of some classifiers, the dataset was enhanced by adding translations of the sentences through an intermediate language back to English. This method was shown to work in a previous Kaggle competition, 'Toxic Comments' \cite{Jigsaw2017}. This is a different application however as it categorises based on the sentiment whereas this challenge is based on the stylistic properties of a given author.
+  
+  The thought behind doing this though, is as there are very few sentences currently, we can get more data to perform analysis on. To do this \textit{TextBlob} was used to translate the sentences in to German, Spanish and French and then back from each respective language into English. 
+  
+  Although an issue may be that some style lost in translation, it appears as though there is some enhancement in the loss by using translations of around 0.03 with the Bag-of-words Feature Kernal Neural Net as seen in the Table \ref{tab:bow_nn_res_tran}. The translation may also help prevent the over fitting that is seen without it.
+  
+\begin{table}[h]
+\centering
+\begin{tabular}{m{1cm}|m{1cm}|m{1cm}|m{1cm}|m{0cm}}
+\multicolumn{1}{m{1cm}}{} & \multicolumn{1}{m{1cm}}{EAP} & \multicolumn{1}{m{1cm}}{HPL} & \multicolumn{1}{m{1cm}}{MWS} &\\[5pt]
+\cline{2-4}
+EAP & 0.82561218 & 0.12039488 & 0.05399294 & \\[5pt]
+\cline{2-4}
+HPL & 0.06722222 & 0.84674603 & 0.08603175 & \\[5pt]
+\cline{2-4}
+MWS & 0.05424431 & 0.1199793 & 0.8257764 & \\[5pt]
+\cline{2-4}
+\end{tabular}
+\caption{Results for bag-of-words feature kernel neural net, stemming and lemmatisation enabled, using a translated enhanced data set.\\Loss 0.43 Accuracy: 83\% }
+\label{tab:bow_nn_res_tran}
+\end{table}
 
-\subsection{KPCA}
-\label{sec:kpca}
+  \subsection{kPCA}
+  \label{sec:kpca}
+    An attempt was made to use kernel principal component analysis (kPCA) to help to produce more separable data. To do this a kPCA was used on a Count Vectoriser and a Term Frequency – Inverse Document Frequency Vectoriser. The idea behind this is to try to make the data points more separable. Although a variety of vectorisers and kernels were used, none were able to improve the separation of data significantly.
```

Comment:
You say "This is to produce more separable data" twice.

This also conflicts with other parts of the report. The kernel bag of words NN uses the same features (term frequency) to separate the data to our best score in the entire project, so it doesn't make sense to say that you couldn't do it here.

If you're going to bring up kPCA and why it didn't work then you need a better reason, because right now this is conflicting.